pages/docs/learn/inngest-functions.mdx

```diff
@@ -0,0 +1,104 @@
+import IconConcurrency from 'src/shared/Icons/FlowControl/Concurrency';
+
+export const description = 'Learn what Inngest functions are and of what they are capable.';
+
+# Inngest Functions
+
+Inngest functions enable developers to create and manage event-driven workflows. They offer a flexible way to build and manage event-driven workflows and background tasks, providing robust tools for retrying, scheduling, and coordinating complex sequences of operations.
+
+This page covers components of an Inngest function, as well as introduces different kinds of functions. If you'd like to learn more about Inngest's execution model, check the ["How Inngest functions are executed"](/docs/learn/how-functions-are-executed) page.
+
+## Anatomy of an Inngest function
+
+Let's have a look at the following Inngest function:
+
+```ts
+import { inngest } from "./client";
+
+export default inngest.createFunction(
+    // config
+  { id: "import-product-images" },
+    // trigger (event or cron)
+  { event: "shop/product.imported" },
+    // handler function
+    async ({ event, step }) => {
+      // here goes the business logic
+      return copyAllImagesToS3(event.data.imageURLs);
+      // if you need to perform more than one task,
+      // wrap each one in step.run()
+    };
+);
+```
+
+The above code can be translated as:
+
+> This Inngest function is called `import-product-images`. When an event called `shop/product.imported` is received, run a function called `copyAllImagesToS3`.
+
+Let's have a look at each of this function's components.
+
+<Callout>
+💡 You can test Inngest functions using standard tooling such as Jest or Mocha. To do so, export the job code and run standard unit tests.
+</Callout>
+
+### Config
+
+The first parameter of the `createFunction` method specifies Inngest function's configuration. In the above example, the `id` is specified, which will be used to identify the function in the [Inngest Dev Server](/docs/local-development):
+
+<img
+  src="/assets/docs/local-development/dev-server-functions.png"
+  width="400" height={400/2762*1868} quality="95"
+  className="rounded-md"
+  alt="Screenshot of the Inngest Dev Server interface showing three functions listed under the 'Functions' tab. The functions are: 'store-events,' 'Generate monthly report,' and 'Customer Onboarding,' each with their respective triggers and App URLs."
+/>
+
+You can also provide other [configuration options](/docs/reference/functions/create#configuration), such as `concurrency`, `throttle`, `debounce`, `rateLimit`, `priority`, `batchEvents`, or `idempotency` (preventing duplicate events from triggering the function within 24 hours). You can also specify how many times the function will retry, what callback function will run on failure, and when to cancel the function.
```

Comment:
**suggestion**: leave out the idempotency explainer and instead link to the flow control overview page?